src/puppetlabs/puppetdb/utils.clj

```diff
@@ -13,7 +13,8 @@
             [slingshot.slingshot :refer [try+ throw+]]
             [com.rpl.specter :as sp])
   (:import [java.net MalformedURLException URISyntaxException URL]
-           [org.postgresql.util PGobject]))
+           [org.postgresql.util PGobject]
+           [puppetlabs.puppetdb.cheshire RawJsonString]))
```

Comment:
sorry, leave out the `[]` it should just be `puppetlabs.puppetdb.cheshire.RawJsonString`, https://clojuredocs.org/clojure.core/import